<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registryForCompanies</class>
 <widget class="QDialog" name="registryForCompanies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#registryForCompanies{
background-color: white ; 
background-image: url(:/images/images/dotted.png); 
}</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat:no-repeat;
background-image: url(:/images/travelSphere.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="RightFrame">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>531</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
background-image: url(:/images/images/blueFrame.png) ; 
border-radius: 15px ; 
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>451</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_2{
background-image: url(:/images/images/innerframe.png) ; 
border-radius: 15px ; 
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>341</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_3{
background-image: url(:/images/images/Liberty.png) ;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="register_PushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="termsAndConditions">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>I agree to the terms and conditions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_loginPage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>670</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to Login Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
  </widget>
  <widget class="QFrame" name="LeftFrame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>581</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_4{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(123, 193, 215, 1), stop:1 rgba(126, 174, 218, 75));
border-radius:50px;
}
#QLineEdit{
border-radius:10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="CompanyName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Company Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="location">
       <property name="placeholderText">
        <string>Established Location</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="note">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#note{
color: #1877F2 

}</string>
    </property>
    <property name="text">
     <string>Register Your Company</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="Nation">
       <property name="placeholderText">
        <string>Nation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="documentNo">
       <property name="placeholderText">
        <string>Documentation No:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="placeholderText">
        <string>Documentation Type</string>
       </property>
       <item>
        <property name="text">
         <string>Citizenship</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Driving License</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Birth Certificate</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ContactNo">
       <property name="placeholderText">
        <string>Company Contact No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CompanyEmail">
       <property name="placeholderText">
        <string>Company Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Company_Username">
       <property name="placeholderText">
        <string>Company Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordComp">
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="establishedDate_2">
       <property name="text">
        <string>Established Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="Established_dateEdit_2">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
